pkg/grpc/api.go

```diff
@@ -299,6 +302,7 @@ func NewAPI(
 		appChannel:                 appChannel,
 		pubsubAdapter:              pubsubAdapter,
 		stateStores:                stateStores,
+		workFlows:                  workfFlows,
```

Comment:
Typo: `workfFlows`, also should it be `workflows`? (or maybe `workflowEngines`?)

Reply:
Updated name to be workflowEngines